server/executor/outputs_processor.go

```diff
@@ -66,7 +66,7 @@ func outputProcessor(ctx context.Context, outputs model.OrderedMap[string, model
 		value := ""
 		spans.
 			ForEach(func(_ int, span traces.Span) bool {
-				value = extractAttr(traces.Span{}, stores, out.expr)
+				value = extractAttr(span, stores, out.expr)
```

Comment:
I tried creating an output, but it was always coming as empty. Then I noticed we were passing an empty span instead of using the one selected by the selector.